.circleci/config.yml

```diff
@@ -0,0 +1,70 @@
+# Use the latest 2.1 version of CircleCI pipeline process engine.
+# See: https://circleci.com/docs/configuration-reference
+version: 2.1
+
+orbs:
+  # Check https://circleci.com/developer/orbs/orb/circleci/go for more details
+  go: circleci/go@1.7.2
+
+commands:
+  nuodb_insight_command:
+    steps:
+      - checkout
+      - go/install:
+          version: "1.20"
+      - run:
+          name: Install Dependencies
+          command: |
+            BASE_COMMIT=$(git merge-base HEAD master)
+            MODIFIED_FILES=$(git diff --name-only "${BASE_COMMIT}")
+            if ! echo "${MODIFIED_FILES}" | grep -qvE '(\.md$)'; then
+              echo "Only docs were updated, stopping build process."
+              exit
+            fi
+            sudo apt-get update
+            sudo apt-get install socat
+            scripts/ci/install_deps.sh
+          environment:
+            GOTESTSUM_VERSION: "1.7.0"
+            TEST_RESULTS: /tmp/test-results
+
+      - run:
+          name: Run tests
+          command: |
+            scripts/ci/determine_and_run_tests.sh
+      - store_test_results:
```

Comment:
The only stores the Junit XML reports. We need to also store the `results` directory created by the test framework.
I think that it is created in the current directory so you could store it via:

```
- store_artifacts:
          path: ./results/
```

Reply:
Added the required changes